apps/examples/testcase/le_tc/network/Make.defs

```diff
@@ -54,73 +54,86 @@
 ifeq ($(CONFIG_EXAMPLES_TESTCASE_NETWORK),y)
 CSRCS += network_tc_main.c
 ifeq ($(CONFIG_TC_NET_SOCKET),y)
-CSRCS +=tc_net_socket.c
+CSRCS += tc_net_socket.c
```

Comment:
Don't fix different kinds of changes. This is a change of formatting (cosmetic).